middleware/chi/example_test.go

```diff
@@ -0,0 +1,41 @@
+package chi_test
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	metrics "github.com/slok/go-http-metrics/metrics/prometheus"
+	"github.com/slok/go-http-metrics/middleware"
+	chiMiddleware "github.com/slok/go-http-metrics/middleware/chi"
+)
+
+// ChiMiddleware shows how you would create a default middleware
+// factory and use it to create a chi compatible middleware.
+func Example_chiMiddleware() {
```

Comment:
ExampleChiMiddleware?

Reply:
made in the same style as everywhere: `Example_fasthttpMiddleware` `Example_stdMiddleware`